governance/votes/Executive Vote - April 29, 2022.md

```diff
@@ -0,0 +1,74 @@
+	- **46,836 DAI** at [0x99E1696A680c0D9f426Be20400E468089E7FDB0f](https://etherscan.io/address/0x99E1696A680c0D9f426Be20400E468089E7FDB0f).
+	- **26,390 DAI** at [0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9](https://etherscan.io/address/0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9).
+- **149,692 DAI** will be transferred to the [Events Core Unit (EVENTS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP32) at [0x3D274fbAc29C92D2F624483495C0113B44dBE7d2](https://etherscan.io/address/0x3D274fbAc29C92D2F624483495C0113B44dBE7d2) as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmSVzGcm#poll-detail).
+- **35,000 DAI** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at [0x955993Df48b0458A01cfB5fd7DF5F5DCa6443550](https://etherscan.io/address/0x955993Df48b0458A01cfB5fd7DF5F5DCa6443550) as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmVdxBE5#poll-detail).
+
+### Core Unit Budget Streams
+
+- **7,590,000 DAI** will be streamed to the [Protocol Engineering Core Unit (PE-001)](https://mips.makerdao.com/mips/details/MIP39c2SP7) at [0xe2c16c308b843eD02B09156388Cb240cEd58C01c](https://etherscan.io/address/0xe2c16c308b843eD02B09156388Cb240cEd58C01c) starting on 2022-05-01 and ending on 2023-04-30 as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmWphKJY#poll-detail).
+- **336,672 DAI** will be streamed to the [Governance Communications Core Unit (COM-001)](https://mips.makerdao.com/mips/details/MIP39c2SP30) at [0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9](https://etherscan.io/address/0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9) starting on 2022-07-01 and ending on 2022-12-31 as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmWQBUi6#poll-detail).
+- **1,083,000 DAI** will be streamed to the [Data Insights Core Unit (DIN-001)](https://mips.makerdao.com/mips/details/MIP39c2SP31) at [0x7327Aed0Ddf75391098e8753512D8aEc8D740a1F](https://etherscan.io/address/0x7327Aed0Ddf75391098e8753512D8aEc8D740a1F) starting on 2022-05-01 and ending on 2023-04-30 as a result of [their most recent budget approval](https://vote.makerdao.com/polling/Qme3VaY4#poll-detail).
+- **748,458 DAI** will be streamed to the [Events Core Unit (EVENTS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP32) at [0x3D274fbAc29C92D2F624483495C0113B44dBE7d2](https://etherscan.io/address/0x3D274fbAc29C92D2F624483495C0113B44dBE7d2) starting on 2022-05-01 and ending on 2023-04-30 as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmSVzGcm#poll-detail).
+
+### DssVest and ESM Interaction
+
+_This is only an announcement. This is not being voted on_.
```

Comment:
Well, it is still a part of the vote. Maybe "_The following sections clarifies the need for the previous vesting streams to take place on a new DssVest contract_"